src/pages/patientView/PatientViewPage.tsx

```diff
@@ -43,6 +45,7 @@ interface IPatientViewState {
     mutationData: any;
     mrnaExprRankData?: MrnaRankData;
     activeTabKey: number;
+    hotspotsData: any;
```

Comment:
@adamabeshouse @alisman just realized that I already have a type defined `IHotspotData`, I just forgot to use that type in `PatientViewPage`. Looks like this PR is already merged, I will create another PR with some other minor improvements as well.